src/messages/src_authority.rs

```diff
@@ -45,6 +57,7 @@ impl SrcAuthority {
     pub(crate) fn src_location(&self) -> SrcLocation {
         match self {
             Self::Node { public_key, .. } => SrcLocation::Node(name(public_key)),
+            Self::BlsShare { public_key, .. } => SrcLocation::Node(name(public_key)),
```

Comment:
is this `mapping` really wanted ?

Reply:
I think this is fine. It's still a single node sending the message, just using a different signature scheme. So `SrcLocation::Node` seems a good choice to me. Don't think we need a one-to-one mapping between `SrcAuthority` and `SrcLocation`.